compiler/src/dotty/tools/dotc/typer/ReTyper.scala

```diff
@@ -48,11 +52,22 @@ class ReTyper extends Typer {
   override def typedSuper(tree: untpd.Super, pt: Type)(implicit ctx: Context): Tree =
     promote(tree)
 
+  override def typedTyped(tree: untpd.Typed, pt: Type)(implicit ctx: Context): Tree = {
+    assertTyped(tree)
+    val type1 = checkSimpleKinded(typedType(tree.tpt))
```

Comment:
Rename `type1 -> tpt1`, `tree1 -> expr1` to conform to naming conventions used elsewhere. (`tpt` stands for "type tree")